Entities/AHuman.cpp

```diff
@@ -4255,11 +4255,7 @@ void AHuman::Update()
 		{
             float rotDiff = rot - (GetRotAngleTarget(m_MoveState) * (m_AimAngle > 0 ? 1 - (m_AimAngle / c_HalfPI) : 1) * GetFlipFactor());
 
-            if (fabs(rotDiff) > 0.1F) {
-                m_AngularVel -= rotDiff * 0.5F;
-            } else if (fabs(m_AngularVel) > 0.3F) {
-                m_AngularVel *= 0.5F;
-            }
+			m_AngularVel = m_AngularVel * 0.9F - (rotDiff * 0.5F);
```

Comment:
I kind of left this half-assed however not on purpose, the reason I changed the original damping is because it was too mild and ultimately resulted in the actor kind of swinging back and forth, which was most prevalent while throwing grenades - as they take torso rotation to account (for whatever reason). The problem I didn't bother to solve was determining the conditions where the RotAngle and AngularVel should be hard-set to 0/whatever, tho I guess I could go for a slightly smaller float. It would only matter in gigantic torsos anyway, where the smallest rotation is noticeable.

Reply:
No worries if you don't wanna deal with hard-setting to 0 in this set of work, it can wait.